.travis.yml

```diff
@@ -0,0 +1,13 @@
+sudo: required
+language: python
+services:
+  - docker
+cache: pip
+python:
+  - "3.6"
+before_install:
+  - python3 -m pip install -U flake8
```

Comment:
any particular reason for splitting this into before_install?

Reply:
How travis likes it :)